File: docs/source/_src_snippets/simulation/environment_setup/environment_models/cannonball_radiation_pressure.py
# Create radiation pressure settings
 reference_area_radiation = 4.0
 radiation_pressure_coefficient = 1.2
 occulting_bodies = [ "Earth" ]

 radiation_pressure_settings = environment_setup.radiation_pressure.cannonball(
     "Sun", reference_area_radiation, radiation_pressure_coefficient, occulting_bodies )

 environment_setup.add_radiation_pressure_interface(
             bodies, "Delfi-C3", radiation_pressure_settings );

File: docs/source/_src_snippets/simulation/environment_setup/environment_models/cannonball_radiation_pressure.cpp
std::string sourceBody = "Sun";

double area = 20.0;
const double radiationPressureCoefficient = 1.2;

std::vector< std::string > occultingBodies;
occultingBodies.push_back( "Earth" );

bodySettings[ "TestVehicle" ]->radiationPressureSettings[ sourceBody ] = std::make_shared< CannonBallRadiationPressureInterfaceSettings >(
    sourceBody, area, radiationPressureCoefficient, occultingBodies );